lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -279,8 +280,8 @@ def main():
         supports_check_mode=True
     )
 
-    if not mysqldb_found:
-        module.fail_json(msg="The MySQL-python module is required.")
+    if mysql_driver is None:
+        module.fail_json(msg="The PyMySQL or MySQL-python module is required.")
```

Comment:
Should this maybe be: `"The PyMySQL or MySQL-python (soon to be deprecated) module is required."`

Also see https://github.com/ansible/ansible/pull/40123/files#r217887801.

Reply:
That suggests PyMySQL doesn't work on Python 2.X at all, which in fact (according to PyMySQL) it doesn't work on <=2.6. So maybe: `The PyMySQL (Python >= 2.7) or MySQL-python (Python 2.X) module is required.` ?

Reply:
How about `The PyMySQL (Python 2.7 and Python 3.X) or MySQL-python (Python 2.X) module is required.` to emphasize that only PyMySQL will work fine with Python 3?

Reply:
:+1: :+1: :+1: